beetsplug/scrub.py

```diff
@@ -101,6 +102,34 @@ def scrub_func(lib, opts, args):
 
         return [scrub_cmd]
 
+    def _preserve(self, path, tags):
+        """Save tags mentioned in tags white list.
+        """
+        self._preservedTags = None
+
+        try:
+            mf = mediafile.MediaFile(path, config['id3v23'].get(bool))
+        except IOError as exc:
+            self._log.error(u'could not open file to preserve tags on scrub: {0}', exc)
+
+        self._preservedTags = dict.fromkeys(tags)
+        for tag in tags:
+            self._preservedTags[tag] = getattr(mf, tag)
+
+    def _restore(self, path):
+        """Restore tags mentioned in tags white list.
+        """
+        if not self._preservedTags:
+            return
+
+        self._log.debug(u'writing new tags after scrub')
+        mf = mediafile.MediaFile(path, config['id3v23'].get(bool))
+        for tag, value in self._preservedTags.items():
+            if value:
+                self._log.info(u'restoring {0} = {1}'.format(tag, value))
```

Comment:
Something is wrong on this line. Not sure what it is, but if I remove this all my tests were successful.